<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionBar</class>
 <widget class="QWidget" name="ActionBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ActionBar {
    background-color: #FFFFFF;
    border-radius: 6px;
}

QPushButton.outlineButton {
    background-color: #EBF5FB;
    color: #3498DB;
    border: 1px solid #BFDBFE;
    border-radius: 4px;
    padding: 8px 15px;
    font-size: 14px;
}

QPushButton.outlineButton:hover {
    background-color: #D1E6FA;
}

QLabel#selectionLabel {
    color: #64748B;
    font-size: 16px;
    background-color: transparent;
}

QPushButton#uploadButton {
    background-color: #2ECC71;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
}

QPushButton#uploadButton:hover {
    background-color: #27AE60;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QPushButton" name="selectAllButton">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="class" stdset="0">
      <string>outlineButton</string>
     </property>
     <property name="text">
      <string>Chọn tất cả</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deselectAllButton">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="class" stdset="0">
      <string>outlineButton</string>
     </property>
     <property name="text">
      <string>Bỏ chọn tất cả</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="selectUnuploadedButton">
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>40</height>
      </size>
     </property>
     <property name="class" stdset="0">
      <string>outlineButton</string>
     </property>
     <property name="text">
      <string>Chọn video chưa tải lên</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="selectionLabel">
     <property name="text">
      <string>Đã chọn: 2 video</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="uploadButton">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Tải lên</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
